src/schema/schema.graphql

```diff
@@ -2884,8 +2884,8 @@ type Mutation {
 	sendInviteReply(inviteReply: InviteReplyInput!): InviteReplyResponse
 	sendShareNotification(shareNotification: ShareNotificationInput!): Boolean
 	setMailboxMetadata(section: String, attrs: MailboxMetadataSectionAttrsInput!): Boolean
-	snoozeCalendarItem(appointment: SnoozeInput, task: SnoozeInput): Boolean,
-	dismissCalendarItem(appointment: DismissInput, task: DismissInput): Boolean,
+	snoozeCalendarItem(appointment: [SnoozeInput], task: SnoozeInput): Boolean,
```

Comment:
should we do similar changes for tasks as well? if not very huge

Reply:
Currently, Reminders are only implemented for appointments and not for Tasks.